routes/chat.js

```diff
@@ -23,7 +23,8 @@ const {
   groupAddMembers,
   removeGroupMember,
   initChatFromProfileAsAnonymousV2,
-  initChatFromProfileAsSignedV2
+  initChatFromProfileAsSignedV2,
+  sendAnonymousMessage
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `sendAnonymousMessage` has been added. Ensure that the function is correctly exported from the `../controllers/chat` module and it handles all necessary error cases to prevent any unhandled exceptions.
```diff
+  sendAnonymousMessage
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->